### PR TITLE
implemented palindrome and tests

### PR DESCRIPTION
**Pull request by argoodm**

### CODE REVIEW

File: functions/palindrome_number.py [hunk 1]
```diff
@@ -1,2 +1,7 @@
-def palindrome_number(x):
-    pass
+def palindrome_number(s):
+    reverse_s = s[::-1]
+
+    if reverse_s == s:
+        return True
+    else:
+        return False
```
Comment (scrambledmegs) [inline functions/palindrome_number.py:7]: Great conditional statement!

File: tests/palindrome_number_test.py [hunk 2]
```diff
@@ -1,2 +1,10 @@
 import pytest
 from functions.palindrome_number import palindrome_number
+
+def test_string_palindrome():
+    s = "mom"
+    
+    reverse_s == palindrome(s)
+    # output : mom
+    
+    assert reverse_s == True
```
Comment (amandajones1996) [inline tests/palindrome_number_test.py:10]: Great edge case test!

**Review by amandajones1996 — COMMENTED** [review]
See comments:

**Review by scrambledmegs — APPROVED** [review]
:)

File: functions/palindrome_number.py (hunk 1 above)
Comment (staceyjo) [inline functions/palindrome_number.py:7]: Could remove , because it will automatically return False if not True

**Review by staceyjo — COMMENTED** [review]
Great work Abby!
